tests/disaster-recovery/sc_arbiter/conftest.py

```diff
@@ -0,0 +1,165 @@
+                )
+                items.remove(item)
+
+
+@pytest.fixture()
+def logwriter_cephfs_many_pvc_factory(request, pvc_factory):
+    def factory(project_name):
+        return pvc_factory(
+            interface=constants.CEPHFILESYSTEM,
+            project=project_name,
+            size="10",
+            access_mode=constants.ACCESS_MODE_RWX,
+        )
+
+    return factory
+
+
+@pytest.fixture()
+def logwriter_workload_factory(request, teardown_factory):
+    def factory(pvc, logwriter_path):
+
+        dc_data = templating.load_yaml(logwriter_path)
+        dc_data["metadata"]["namespace"] = pvc.namespace
+        dc_data["spec"]["replicas"] = 4
+        dc_data["spec"]["template"]["spec"]["containers"][0][
+            "image"
+        ] = "quay.io/ocsci/logwriter:latest"
+        dc_data["spec"]["template"]["spec"]["volumes"][0]["persistentVolumeClaim"][
+            "claimName"
+        ] = pvc.name
+        logwriter_dc = helpers.create_resource(**dc_data)
+        teardown_factory(logwriter_dc)
+
+        logwriter_dc_obj = Deployment(
+            **get_deployments_having_label(
+                label="app=logwriter-cephfs", namespace=pvc.namespace
+            )[0]
+        )
+        logwriter_dc_pods = [
+            pod["metadata"]["name"]
+            for pod in get_pods_having_label(
+                label="app=logwriter-cephfs", namespace=pvc.namespace
+            )
+        ]
+        wait_for_pods_to_be_running(
+            namespace=pvc.namespace, pod_names=logwriter_dc_pods
+        )
+
+        return logwriter_dc_obj
+
+    return factory
+
+
+@pytest.fixture()
+def logreader_workload_factory(request, teardown_factory):
+    def factory(pvc, logreader_path, duration=30):
+
+        job_data = templating.load_yaml(logreader_path)
+        job_data["metadata"]["namespace"] = pvc.namespace
+        job_data["spec"]["completions"] = 4
+        job_data["spec"]["parallelism"] = 4
+        job_data["spec"]["template"]["spec"]["volumes"][0]["persistentVolumeClaim"][
+            "claimName"
+        ] = pvc.name
+        job_data["spec"]["template"]["spec"]["containers"][0][
+            "image"
+        ] = "quay.io/ocsci/logwriter:latest"
+        job_data["spec"]["template"]["spec"]["containers"][0]["command"][
+            2
+        ] = f"/opt/logreader.py -t {duration} *.log -d"
+        logreader_job = helpers.create_resource(**job_data)
+        teardown_factory(logreader_job)
+
+        logreader_job_obj = get_job_obj(
+            name="logreader-cephfs", namespace=pvc.namespace
+        )
+        logreader_job_pods = [
+            pod["metadata"]["name"]
+            for pod in get_pods_having_label(
+                label="app=logreader-cephfs", namespace=pvc.namespace
+            )
+        ]
+        wait_for_pods_to_be_running(
+            namespace=pvc.namespace, pod_names=logreader_job_pods
+        )
+
+        return logreader_job_obj
+
+    return factory
```

Comment:
These fixtures don't seem to be specific to arbiter. I suggest placing them in tests/conftest.py